src/reusable-blocks/class-blocks.php

```diff
@@ -30,8 +33,9 @@ public function getIdsFromPost( $post_id ) {
 				       && isset( $block['attrs']['ref'] )
 				       && is_numeric( $block['attrs']['ref'] );
 			})->map( function( $block ) {
-				return (int) $block['attrs']['ref'];
-			})->toArray();
+				$block_id = (int) $block['attrs']['ref'];
+				return array_merge( [ $block_id ], $this->getIdsFromPost( $block_id ) );
```

Comment:
@strategio I think the name of the method is a bit unclear. 

What about `getChildrenIdsFromPost` or something like that?

It would make clear to the reader that you are adding the IDs of the nested blocks.

Reply:
Done!